### DISCUSSION TITLE
Add language model when decoding

### DISCUSSION THREAD

**Discussion by zhhuang93** [openai/whisper#868] [Q&A]
Does anyone add language model for a specified domain? How is the performance?

**Comment by hashefa** [comment]
Is there any work in this direction?

**Reply by jmukiibi** [reply]
Could adding a language model improve the model performance on local names (People names, place names...)?

**Comment by SreeramGanji** [comment]
Is there any work in this direction? I actually wanted to test wisper english model with my own language model while decoding. 

Can anyone help me in this regard ?

**Reply by RajGothi** [reply]
Have you found any work around this?
